.gitignore

```diff
@@ -52,6 +53,12 @@ symbols.csv
 res/**/*.c
 *.o
 
+# SDAT Files
+res/sound/*.sdat
+res/sound/*/Files/BANK/*.sbnk
+res/sound/*/Files/SEQ/*.sseq
+res/sound/*/Files/WAVARC/*.swar
```

Comment:
Are these files generated inside the source-tree? If so, can we make a patch to SDATTool to generate them into a specified folder?

Reply:
They are not, we gucci